website/meta/languages.json

```diff
@@ -36,12 +36,7 @@
             "name": "Catalan",
             "example": "Això és una frase.",
             "has_examples": true,
-            "models": [
-                "ca_core_news_sm",
-                "ca_core_news_md",
-                "ca_core_news_lg",
-                "ca_core_news_trf"
-            ]
+            "models": ["ca_core_news_sm", "ca_core_news_md", "ca_core_news_lg", "ca_core_news_trf"]
```

Comment:
Again, wondering why the formatting has to change?

Reply:
Basically because "Prettier says so". I just ran it to apply to all files that it would apply to. Anyone who has the Prettier extension installed with fix-on-save would get the same result when opening and saving this file.